www-servers/lighttpd/lighttpd-1.4.59.ebuild

```diff
@@ -0,0 +1,250 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} )
+
+inherit autotools flag-o-matic lua-single readme.gentoo-r1 systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="Lightweight high-performance web server"
+HOMEPAGE="https://www.lighttpd.net https://github.com/lighttpd"
+SRC_URI="https://download.lighttpd.net/lighttpd/releases-1.4.x/${P}.tar.xz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="brotli bzip2 dbi doc gdbm gnutls ipv6 kerberos ldap libev lua maxminddb minimal mbedtls mmap memcached mysql nss pcre php postgres rrdtool sasl selinux ssl sqlite test webdav xattr zlib zstd"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )
+	minimal? (
+		!brotli
+		!bzip2
+		!doc
+		!lua
+		!maxminddb
+		!pcre
+		!rrdtool
+		!webdav
+		!zlib
+	)
```

Comment:
Yea no this is unacceptable, if you really want to introduce the minimal flag, you should do it purely in `src_configure` and `pkg_postinst`.

Reply:
I don't see why we need it. I was just afraid to rip it out. Seems to be all controlled by the USE flags now, as I was told in the previous pull request. I've removed it now.